consensus/types/src/eth_spec.rs

```diff
@@ -394,8 +419,12 @@ impl EthSpec for MainnetEthSpec {
     type MaxBlobCommitmentsPerBlock = U4096;
     type BytesPerFieldElement = U32;
     type FieldElementsPerBlob = U4096;
+    type FieldElementsPerCell = U64;
+    type FieldElementsPerExtBlob = U8192;
     type BytesPerBlob = U131072;
+    type BytesPerCell = U2048;
     type KzgCommitmentInclusionProofDepth = U17;
+    type KzgCommitmentsInclusionProofDepth = U4; // inclusion of the whole list of commitments
```

Comment:
Just noting that the naming here is really confusing (w.r.t `KzgCommitmentInclusionProofDepth`) and has a potential for logic errors. Maybe we can address the naming in the spec